pkg/shell/credentials.jsx

```diff
@@ -171,7 +171,7 @@ const AddNewKey = ({ load, unlockKey, onClose }) => {
                 <FileAutoComplete onChange={setNewKeyPath}
                                   placeholder={_("Path to file")}
                                   superuser="try" />
-                {newKeyPathError && <HelperText className="pf-c-form__helper-text">
+                {newKeyPathError && <HelperText className="pf-v5-c-form__helper-text">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230516-073248-d32f52be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)